src/systems/constraints-system.js

```diff
@@ -0,0 +1,90 @@
+const ACTIVATION_STATE = require("aframe-physics-system/src/constants").ACTIVATION_STATE;
+
+function storeState(prev, curr) {
+  prev.held = curr.held;
+  prev.hovered = curr.hovered;
+  prev.spawning = curr.spawning;
+}
+
+export class ConstraintsSystem {
+  constructor() {
+    this.prevLeftHand = {
+      held: null,
+      hovered: null,
+      spawning: null
+    };
+    this.prevRightHand = {
+      held: null,
+      hovered: null,
+      spawning: null
+    };
+    this.prevRightRemote = {
+      held: null,
+      hovered: null,
+      spawning: null
+    };
+  }
+
+  tickInteractor(options, state, prevState) {
+    if (prevState.held === state.held) {
+      if (
+        state.held &&
+        state.held.components.tags &&
+        state.held.components.tags.data[options.constraintTag] &&
+        prevState.spawning &&
+        !state.spawning
```

Comment:
It would be great to get rid of this `spawning` tag, but I don't know how to do it.